ui/lib/core/addon/components/replication-page.js

```diff
@@ -17,16 +26,10 @@ export default Component.extend({
     }
     return false;
   }),
-  title: computed('model', function() {
-    let mode = this.model.rm.mode;
-    if (mode === 'dr') {
-      return 'Disaster Recovery';
-    } else if (mode === 'performance') {
-      return 'Performance';
-    }
-    return 'unknown';
-  }),
   message: computed('model', function() {
-    return 'This Disaster Recovery secondary has not been enabled.  You can do so from the Disaster Recovery Primary.';
+    if (this.model.anyReplicationEnabled) {
+      return 'This Disaster Recovery secondary has not been enabled.  You can do so from the Disaster Recovery Primary.';
```

Comment:
i think you'll want to use `${this.mode}` here as well so that the message makes sense for Performance secondaries too, right?